doc/lua_api.txt

```diff
@@ -4309,12 +4309,21 @@ Definition tables
 * `"image.png"`
 * `{name="image.png", animation={Tile Animation definition}}`
 * `{name="image.png", backface_culling=bool, tileable_vertical=bool,
-    tileable_horizontal=bool}`
+    tileable_horizontal=bool, align_style="node"/"world"/"user", scale=float}`
     * backface culling enabled by default for most nodes
     * tileable flags are info for shaders, how they should treat texture
 	  when displacement mapping is used
 	  Directions are from the point of view of the tile texture,
 	  not the node it's on
+    * align style determines will the texture be rotated with the node
+          or kept aligned with its surroundings. "user" means that client
+          setting will be used, similar to `glasslike_framed_optional`.
+          Note: supported by solid nodes and nodeboxes only.
+          Note: old syntax `world_aligned=true` is supported as well.
+    * scale is used on world-aligned textures only, making them effectively
+          span several nodes (exactly `scale` nodes, if used properly;
+          as the effect is applied on per-mapblock basis, (16/scale) should
```

Comment:
How about something like `The texture will be split ``scale`` times in each dimension across a mapblock.` ?

Reply:
Not `scale` times, but `scale` nodes, instead of just one.